week-9/Homework/mandatory/2-exercises/1-shopping-cart.js

```diff
@@ -10,10 +10,18 @@ The output of running your code should be:
 */
 
 class ShoppingCart {
-  // Add your code here
+  constructor() {
+    this.total = 0;
+    this.array = [];
+  }
+
+  addItem(item) {
+    this.array.push(item);
+    this.total++;
```

Comment:
This works, but the code would be better if you only had one instance variable `this.array` and instead got the total from a function i.e.
```js
total() {
  return this.array.length;
}
```

This is better because you don't have to keep track of two variables, you only need one.